pkg/layer/encoding/czml.go

```diff
@@ -56,13 +56,26 @@ func (e *CZMLEncoder) encodeSingleLayer(li *merging.SealedLayerItem) (*czml.Feat
 		if li.Property.Field("location") != nil {
```

Comment:
```suggestion
		if f := li.Property.Field("location"); f != nil {
```

More readable code

Change other parts of the file in the same way.